Implement `mkr channels pull` command

stefafafan opened the pull request:
Since mackerel-client-go tag 0.9.0 supports Channels API ( https://github.com/mackerelio/mackerel-client-go/releases/tag/v0.9.0 ), I tried implementing the channels command for mkr.

What I've done:
- Bump mackerel-client-go to 0.9.0 to be able to use `FindChannels`
- Added `channels` command (lists all the channels)
- Added `channels pull` as well (saves result to `channels.json` )

I basically referenced the `mkr monitors` command.

---
To test,  I ran `make cross` to make a build that runs on my mac, and tried running `mkr channels` and `mkr channels pull` using the build. They seemed to be working as I intend.

ne-sachirou commented on channels.go:
```diff
@@ -0,0 +1,88 @@
+	}
+
+	if filePath == "" {
+		filePath = "channels.json"
```
You are assigning the default value of `filePath` at here and in `channelSaveRules`. It may cause bugs in the future. I think if you move this assignment to the top of the L58 then the second assignment is become useless.

ne-sachirou commented:
- Can't you seperate this to a new `channels` module like hosts, services, org, plugin, checks and wrap?
- Can't you write tests?

stefafafan commented:
Thank you for the comments. I referred to the monitors file when writing this, but I will take a look at the newer modules.

stefafafan commented:
Sorry @ne-sachirou for having you wait, I added the following:

- I separated `channels` to its own package
- I added tests to the channels package
- I fixed conflict while merging latest master branch

I also built and re-tested the mkr channel command with my personal Mackerel org and it seemed to still work. Would you be able to take another look at the code?

ne-sachirou commented:
@stefafafan LGTM!

stefafafan commented:
Thank you!!
